person.rb

```diff
@@ -1,8 +1,10 @@
+require_relative 'nameable'
 class Person
```

Comment:
```suggestion
class Person < Nameable
```
Based on the requirements kindly:
- [ ] Make sure that your Person class inherits from Nameable
 
- [ ]  Make sure that this class has a method correct_name implemented. It should simply return the name attribute.


````ruby
 def correct_name
       name
    end
````